chain-evm/src/machine.rs

```diff
@@ -70,6 +70,21 @@ pub type GasPrice = U256;
 /// Gas limit for EVM operations.
 pub type GasLimit = U256;
 
+#[derive(Clone, Default, Debug, PartialEq, Eq)]
+/// EVM Environment parameters needed for execution.
+pub struct Environment {
+    pub gas_price: GasPrice,
+    pub origin: Origin,
+    pub chain_id: ChainId,
+    pub block_hashes: BlockHashes,
+    pub block_number: BlockNumber,
+    pub block_coinbase: BlockCoinBase,
+    pub block_timestamp: BlockTimestamp,
+    pub block_difficulty: BlockDifficulty,
+    pub block_gas_limit: BlockGasLimit,
+    pub block_base_fee_per_gas: BlockBaseFeePerGas,
```

Comment:
This was introduced with [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) in the London HF, changing the way that gas is consumed.

Reply:
Judging from this

> The proposal in this EIP is to start with a base fee amount which is adjusted up and down by the protocol based on how congested the network is.

we can set to `0` or any arbitrary constant as we are not using it. Because there is no fee market in our implementation and fees are static.